tests/README.md

```diff
@@ -0,0 +1,10 @@
+# Tests
+
+## Tags
+Tags are used to filter tests and are indicated by `@<tag name>` at the top of the feature file.
+
+To limit the scope of the test session to the scenarios with tag `@example_tag`, specify the
+filter to tox with `-k`, which will pass it through to the underlying pytest command.
+
+`tox -e tests -- -k "example_tag"`
```

Comment:
Something's off in these 2 lines. First command seems correct, next line missing `-e`, and shouldn't be duplicated of course.